src/components/NavBar.vue

```diff
@@ -18,30 +26,50 @@
 </template>
 
 <script setup>
-import { ref } from 'vue';
+import { ref, computed } from 'vue';
 import { useRouter } from 'vue-router';
 import { storeToRefs } from 'pinia';
 import { useAuthStore } from '@/store/auth';
 import _get from 'lodash/get';
+import {getSidebarActions} from '@/router/sidebarActions'
+ import { fetchDocById } from '@/helpers/query/utils';
+import { useQuery } from '@tanstack/vue-query';
 
 const router = useRouter();
 const authStore = useAuthStore();
 const { roarfirekit } = storeToRefs(authStore);
+const initialized = ref(true);
+const menu = ref();
+
+const { data: userClaims } = useQuery({
+  queryKey: ['userClaims', authStore.uid],
+  queryFn: () => fetchDocById('userClaims', authStore.uid),
+  keepPreviousData: true,
+  enabled: initialized,
+  staleTime: 5 * 60 * 1000, // 5 minutes
+});
+
+ const isSuperAdmin = computed(() => Boolean(userClaims.value?.claims?.super_admin));
+
+
+const isAtHome = computed(()=>{
+  return router.currentRoute.value.fullPath === '/'
+});
+const dropDownActions = computed(() => {
+  const rawActions = getSidebarActions(authStore.isUserSuperAdmin, !isAtHome.value);
```

Comment:
Let's use the computed value that you defined above.
```suggestion
  const rawActions = getSidebarActions(isSuperAdmin.value, !isAtHome.value);
```